Src/Base/AMReX_OpenMP.H

```diff
@@ -17,8 +17,7 @@ namespace amrex::OpenMP {
     void Initialize ();
     void Finalize ();
 
-    static constexpr int nlocks = 128;
-    extern AMREX_EXPORT omp_lock_t omp_locks[nlocks];
+    omp_lock_t* get_lock (int ilock);
```

Comment:
This might need an export though:
```suggestion
    AMREX_EXPORT omp_lock_t* get_lock (int ilock);
```

Reply:
Could this be missing includes? #3802

Reply:
I don't see any missing includes.

Reply:
Let's add a test that uses lockAdd to amrex/Tests/ to see if conda-forge Windows build work for amrex.

Reply:
I wonder how `<omp.h>` and `AMReX_OpenMP.H` come into downstream files such as `ChargeDeposition.cpp` and `BaseFab::lockAdd`. Neither `AMReX_MultiFab.H` nor `AMReX_BaseFab.H` seem to include it directly, must be some transitive path. Anyway, #3802 does some cleaning for directly uses classes for future robustness.